ranger/core/main.py

```diff
@@ -180,6 +180,16 @@ def main(
             for command in args.cmd:
                 fm.execute_console(command)
 
+        if int(os.environ[level]) > 1:
+            warning = 'Warning:'
+            nested_warning = "You're in a nested ranger instance!"
+            nrw = fm.settings.nested_ranger_warning.lower()
```

Comment:
Let's name this variable `warning_level` or actually almost anything that exists in a dictionary…